pkg/apis/objectbucket.io/v1alpha1/objectbucket_types.go

```diff
@@ -22,9 +21,12 @@ type ObjectBucketSpec struct {
 	SecurePort int `json:"securePort"`
 	// SSL true if the connection is secured with SSL, false if it is not.
 	SSL bool `json:"ssl"`
-
+	// BucketACL
```

Comment:
If `SecurePort` has a value does that mean accessing the bucket should be via this port? And, if true, then what is the purpose of the SSL bool?

Reply:
This is derived from some other examples I've seen on bucket CRD implementations.  It's true that it's redundant.